apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts

```diff
@@ -117,6 +117,9 @@ export class PrimarySidebarComponent implements OnInit, OnChanges, OnDestroy {
     public companyList: CompanyResponse[] = [];
     /** Stores all the menu items to be shown */
     public allItems: AllItems[] = [];
+    /** True, if sidebar needs to be shown */
+    public subdropdown;
```

Comment:
define type of variable like boolean/string/number etc....

Reply:
Done